docs/src/Provider.doc.js

```diff
@@ -111,41 +113,39 @@ function OnNavigation() {
       window.open("https://help.pinterest.com", "_blank");
     }
 
-    return onNavigationOptions && onNavigationOptions.navigationMode === "client"
+    return onNavigationOptions && onNavigationOptions.navigationMode === "link"
       ? onNavigationClick
       : null;
   }
 
   const linkProps = {
     href: "https://pinterest.com",
-    onNavigationOptions: {
-      navigationMode: clientOnNavigationMode ? "client" : "server"
-    },
+    onNavigationOptions: { navigationMode: clientOnNavigationMode },
     target: "blank",
   }
 
   return (
     <Provider onNavigation={onNavigation}>
       <Flex direction="column" gap={2}>
-        <Flex direction="row" gap={2}>
-          <Switch
-            onChange={() => setClientOnNavigationMode(!clientOnNavigationMode)}
-            id="disable-buttons"
-            switched={clientOnNavigationMode}
+        <Flex direction="column" gap={2}>
+          <Text>Navigation type:</Text>
+          <RadioButton
+            checked={clientOnNavigationMode === "link"}
+            id="link"
+            label="Default Link"
+            name="navigation"
```

Comment:
Ahh each set of RadioButtons needs a unique name (can't click the 2nd and 3rd sets right now)

Reply:
DONE

Reply:
Sorry just noticed it - all the value="dogs" should be removed 😛

Reply:
DONE, yeah noticed it too